wlog/svclog/svc1log/logger_default.go

```diff
@@ -15,9 +15,24 @@
 package svc1log
 
 import (
+	"time"
+
 	"github.com/palantir/witchcraft-go-logging/wlog"
 )
 
+var DebugLevelParam = wlog.NewParam(func(entry wlog.LogEntry) {
```

Comment:
What's the high-level motivation/explanation for why the logger level is now a param?

If we were to actually do this, then these should probably be functions that return the param -- it's not good practice to have exported package-level variables since they are mutable (anyone importing the package could do something like `svc1log.DebugLevelParam = nil` and reassign the global value)